data/json/item_groups.json

```diff
@@ -6850,7 +6850,8 @@
     "id" : "helicopter",
     "magazine": 100,
     "items":[
-      { "group": "guns_launcher_milspec", "prob": 10 },
+      { "group": "guns_launcher_milspec", "prob": 20 },
+      { "group": "guns_launcher_rocket", "prob": 44 },
```

Comment:
Aren't changes a bit high?

Reply:
The guns_launcher_milspec spawns are a bit high. The launcher_rocket spawn chance is a total of the original spawn chances for a launcher or launcher ammo in this group:

```

      ["LAW_Packed", 30],
      ["m3_carlgustav", 1],
      ["84x246mm_he", 3],
      ["84x246mm_hedp", 2],
      ["84x246mm_smoke", 3],
      ["m202_flash", 2],
      ["m235tpa", 3],

```